### ISSUE TITLE
About train dataset.

### ISSUE THREAD

**Issue by BlossomingL** [DAA233/generative-inpainting-pytorch#20]
Hi~
What train dataset did you use? I mean in ImageNet2012 there are many sub-datasets, I want to know which did you use?

**Comment by DAA233** [comment]
I used all the training images in ImageNet 2012 training set (in total 1000 classes) and evaluated on the validation set with a center crop mask.

**Comment by BlossomingL** [comment]
Thanks for your quick reply, the dataset is about 137GB or more? So big....
